bench.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+import os
+import sys
+import time
+
+import click
+import statistics
+
+
+def calculator(test, arg, n_runs, only_result, books=False):
+    w = []
+    for x in range(n_runs):
+        if only_result:
+            process = subprocess.Popen(
+                arg,
+                shell=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE)
+            process.wait()
+        else:
+            process = subprocess.Popen(arg, shell=True)
+            process.wait()
+        with open('Benchmark.txt') as f:
+            for line in f.readlines():
+                w.append(float(line))
+
+    click.secho(
+        "\nThe results of the benchmark are (all speeds in items/sec) : \n",
+        bold=True)
+    click.secho(
+        "\nTest = '{0}' Iterations = '{1}'\n".format(test, n_runs),
+        bold=True)
+    click.secho(
+        "\nMean : {0} Median : {1} Std Dev : {2}\n".format(
+            statistics.mean(w),
+            statistics.median(w),
+            statistics.pstdev(w)),
+        bold=True)
+    os.remove('Benchmark.txt')
+    try:
+        os.remove('items.csv')
```

Comment:
Maybe it's better to move this to ``bookworm`` function? Then you would not need a try/except here